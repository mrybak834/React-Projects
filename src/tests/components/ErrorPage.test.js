import React from 'react';
import { shallow } from 'enzyme';
import ErrorPage from '../../components/ErrorPage';

test('Components: ErrorPage: Render Error Page', () => {
    const wrapper = shallow(<ErrorPage />);

    expect(wrapper).toMatchSnapshot();
});