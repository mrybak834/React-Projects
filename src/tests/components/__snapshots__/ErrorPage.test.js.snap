// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components: ErrorPage: Render Error Page 1`] = `
<div>
  404! - 
  <Link
    replace={false}
    to="/"
  >
    Go home
  </Link>
</div>
`;
